pkg/replicassyncer/utils.go

```diff
@@ -0,0 +1,72 @@
+package replicassyncer
+
+import (
+	"context"
+
+	autoscalingv1 "k8s.io/api/autoscaling/v1"
+	autoscalingv2 "k8s.io/api/autoscaling/v2"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/scale"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+)
+
+const KindHPA = "HorizontalPodAutoscaler"
```

Comment:
We can use [it](https://github.com/karmada-io/karmada/blob/8d1a8007ebdbd97a2fe12cf8ee7d4694cfb2ed82/pkg/util/constants.go#L108) to replace this const

Reply:
No more need after deprecated the webhook.